SimplenoteUITests/NoteEditor.swift

```diff
@@ -34,6 +34,17 @@ class NoteEditor {
             noteContentTextView.clearText()
             noteContentTextView.paste(text: enteredValue)
 
+            // Once the text is pasted, there might be an info bubble shown at the note top,
+            // saying "Simplenote pasted from Simplenote-UITestsRunner". Since it covers all
+            // NavBar buttons, we have to wait for it to disappear:
+            let isHittablePredicate = NSPredicate { _, _ in
+                app.buttons[UID.Button.dismissKeyboard].isHittable == true
+            }
+
+            let expectation = XCTNSPredicateExpectation(predicate: isHittablePredicate, object: .none)
+            XCTWaiter().wait(for: [expectation], timeout: averageLoadTimeout)
+            // end of wait
```

Comment:
Nitpick: Unlike the comment above about the bubble blocking the navigation bar button, which is super useful, this comment doesn't add much information:

```suggestion
```